packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -79,20 +86,17 @@ def config_all_loggers():
     loggers = [logging.getLogger()] + [
         logging.getLogger(name) for name in the_manager.loggerDict
     ]
+
     for logger in loggers:
-        set_logging_handler(logger)
+        set_logging_handler(logger, DEFAULT_FORMATTING)
 
 
 def set_logging_handler(
     logger: logging.Logger,
-    formatter_base: Optional[type[logging.Formatter]] = None,
-    fmt: str = DEFAULT_FORMATTING,
+    fmt: str,
```

Comment:
should fit on one line
```suggestion
    fmt: str
```